include/openssl/hpke.h

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright 2019-2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/**
+ * @file
+ * Data structures/prototypes for HPKE (RFC9180)
+ */
+
+#ifndef HPKE_H_INCLUDED
+# define HPKE_H_INCLUDED
```

Comment:
Following the naming scheme we already have for these guards, it should be `OPENSSL_HPKE_H`

Reply:
fixed now ta